onlineweb4/settings/base.py

```diff
@@ -1,28 +1,39 @@
 # -*- coding: utf8 -*-
 import os
 import sys
-from webpack_resolve import create_resolve_file
 
+import dj_database_url
+from decouple import config
 from django.contrib.messages import constants as messages
+from webpack_resolve import create_resolve_file
 
 # Directory that contains this file.
 PROJECT_SETTINGS_DIRECTORY = os.path.dirname(globals()['__file__'])
 # Root directory. Contains manage.py
 PROJECT_ROOT_DIRECTORY = os.path.join(PROJECT_SETTINGS_DIRECTORY, '..', '..')
 
-TEST_RUNNER = "django_nose.NoseTestSuiteRunner"
+sys.dont_write_bytecode = config("OW4_PYTHON_DONT_WRITE_BYTECODE", cast=bool, default=True)
```

Comment:
Copied it from example-local.py. No idea. Is it a Python 2.x thing? We could drop it I guess.